GsaGH/Parameters/5_Results/VectorResultGoo.cs

```diff
@@ -12,9 +12,10 @@ namespace GsaGH.Parameters
   /// </summary>
   public class VectorResultGoo : GH_GeometricGoo<GH_Vector>, IGH_PreviewData
   {
-    public readonly Point3d StartingPoint;
+    public Point3d StartingPoint;
```

Comment:
Starting Point is always the node position, and we don't want to make it possible to change it outside this class.

Reply:
users can move the results?